fuzon-http/src/main.rs

```diff
@@ -132,3 +135,21 @@ async fn main() -> std::io::Result<()> {
     .run()
     .await
 }
+
+fn main() -> Result<()> {
+    // TODO: This is unsafe: not sure what to do though -> maybe block the executor and set env
+    // variables before this asyn function is started.
+    // Strange that Rust compiled anyway, without the unsafe block, any idea?
+    unsafe {
+        env::set_var(
+            "RUST_LOG",
+            "fuzon_http=info,actix_web=warn,actix_server=info",
+        );
+    }
+
```

Comment:
**question:** how is this unsafe? If I remove the unsafe directive, everything works fine. This function is in a sync context and we only start the runtime after exporting, right?